pkg/k3s/build.yml

```diff
@@ -0,0 +1,24 @@
+# linuxkit build template 
+#
+# Copyright (c) 2023 Zededa, Inc.
+# SPDX-License-Identifier: Apache-2.0
+org: lfedge
+image: eve-k3s
+config:
+  binds:
+    - /lib/modules:/lib/modules
+    - /dev:/dev
+    - /etc/resolv.conf:/etc/resolv.conf
+    - /run:/run
+    - /config:/config
+    - /:/hostfs
+    - /persist:/persist:rshared,rbind
+    - /sys/fs/cgroup:/sys/fs/cgroup
+  net: host
```

Comment:
We won't run build with network access, you can see similar discussion on why here: https://github.com/lf-edge/eve/pull/299

From @eriknordmark's words:
```
We want to use ADD and run without network since that makes it possible to generate a SBOM for the build.
```

Reply:
Every eve container is build with net: host

bash-3.2$ grep net pkg/*/build.yml

pkg/alpine-base/build.yml:network: yes
pkg/alpine/build.yml:network: yes
pkg/debug/build.yml:  net: host
pkg/edgeview/build.yml:  net: host
pkg/fscrypt/build.yml:# we allow network in building fscrypt because we need it to downlaod go modules,
pkg/fscrypt/build.yml:network: yes
pkg/guacd/build.yml:  net: host
pkg/kdump/build.yml:network: yes
pkg/kvm-tools/build.yml:  net: host
pkg/newlog/build.yml:  net: host
pkg/pillar/build.yml:  net: host
pkg/rngd/build.yml:  net: new
pkg/vtpm/build.yml:  net: host
pkg/wlan/build.yml:  net: host
pkg/wwan/build.yml:  net: host
pkg/xen-tools/build.yml:  net: host

But that is fine I can remove if not needed